cypress/e2e/DiagramsPage/ZoomDiagrams.feature

```diff
@@ -31,7 +31,8 @@ Feature: Test diagrams page: zoom on all diagrams
 
     # Back to the diagrams page
     When I visit the 'localhost:8080/#/projects/{{ projectName }}/diagrams'
-    And I wait 2 seconds
```

Comment:
put this in another commit like `fix e2e zoom test`

Reply:
done